src/css/styles/_general.scss

```diff
@@ -22,3 +23,61 @@
 .row {
   margin: 0;
 }
+
+.top {
+  position: relative;
+  width: 100%;
+  min-height: 500px;
+  overflow: hidden;
+  background: url("../img/banner-bg.jpg") no-repeat 75%;
+  background-size: cover;
+}
+
+.banner {
+  padding-top: 60px;
+}
+
+.repair1,
+.repair2,
+.production1,
+.production2 {
+  margin-bottom: 6rem;
+}
```

Comment:
> I propose, using both- descriptive classes (or ids if better) with updated naming (repair-primary, repair-secondary) and styling class (subsection, because we already have <section> as a tag and c-section as block).

LGTM :)